settings.ini

```diff
@@ -4,7 +4,7 @@ lib_name = fasthtml
 version = 0.0.11
 min_python = 3.10
 license = apache2
-requirements = fastcore>=1.5.45 python-dateutil starlette oauthlib itsdangerous uvicorn httpx fastlite>=0.0.6 python-multipart
+requirements = fastcore>=1.5.45 python-dateutil starlette oauthlib itsdangerous uvicorn[standard] httpx fastlite>=0.0.6 python-multipart sqlite-utils
```

Comment:
I was getting a missing package error for `sqlite_utils` so I added `sqlite-utils`.

Reply:
discussed and resolved [here](https://github.com/AnswerDotAI/fasthtml/pull/9#discussion_r1632399002)